hack/build-bytecode-images.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Set default value for IMAGE_INFW_BC tag if not already set
+IMAGE_INFW_BC=${IMAGE_INFW_BC:-quay.io/bpfman-bytecode/ingress-node-firewall}
+
+# PROGRAMS is a list of <program name>:<program type> tuples
+PROGRAMS="{\
+\"xdp_ingress_node_firewall_process\":\"xdp\",\
+\"tcx_ingress_node_firewall_process\":\"tc\"\
```

Comment:
If the program name is `"tcx_..."`, should the program type be `"tcx"`?

Reply:
will hold on this till bpfman support tcx this more of hack for now